src/main/java/org/jenkinsci/tracey/RabbitMQConnectionHolder.java

```diff
@@ -0,0 +1,16 @@
+package org.jenkinsci.tracey;
+
+import net.praqma.tracey.broker.impl.rabbitmq.RabbitMQConnection;
+
+public class RabbitMQConnectionHolder {
+
+    private static RabbitMQConnection connection;
```

Comment:
How this supposed to work with more than two jobs that have triggers? If understand right then every time you create a new job it will use the same connection or? Or you use the same connection